pkg/cmd/server/origin/run_components.go

```diff
@@ -508,3 +510,31 @@ func (c *MasterConfig) RunClusterQuotaMappingController() {
 		go c.ClusterQuotaMappingController.Run(5, utilwait.NeverStop)
 	})
 }
+
+func (c *MasterConfig) RunClusterQuotaReconciliationController() {
+	osClient, kClient := c.ResourceQuotaManagerClients()
+	resourceQuotaRegistry := quotainstall.NewRegistry(kClient)
+	groupKindsToReplenish := []unversioned.GroupKind{
+		kapi.Kind("Pod"),
+		kapi.Kind("Service"),
+		kapi.Kind("ReplicationController"),
+		kapi.Kind("PersistentVolumeClaim"),
+		kapi.Kind("Secret"),
+		kapi.Kind("ConfigMap"),
```

Comment:
> Are these all we can replenish? Image strems? Deployment configs?

for now, there's an item on the card: https://trello.com/c/qJUVjQnj/713-13-multi-project-quota